scripts/replace-entry.js

```diff
@@ -0,0 +1,14 @@
+const fs = require("fs");
+const path = require("path");
+
+console.log("Copying index.release.js -> index.js ...");
+fs.copyFileSync(
+  path.join(__dirname, "..", "index.release.js"),
+  path.join(__dirname, "..", "index.js")
+);
+
+console.log("Copying index.release.d.ts -> index.d.ts ...");
+fs.copyFileSync(
+  path.join(__dirname, "..", "index.release.d.ts"),
+  path.join(__dirname, "..", "index.d.ts")
+);
```

Comment:
Some of this is also part of `scripts/prepublish`. Makes me wonder if this can be unified / reused

Reply:
I guess we could add some special flag for CI in prepublish script.